extension/persistence/relational-jdbc/build.gradle.kts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+fun isValidDep(dep: String): Boolean {
+  val depRegex = "^[\\w.]+:[\\w\\-.]+:[\\w\\-.]+$".toRegex()
+  return dep.matches(depRegex)
+}
+
+plugins {
+  id("polaris-server")
+  alias(libs.plugins.quarkus)
+  alias(libs.plugins.jandex)
+}
+
+dependencies {
+  implementation(project(":polaris-core"))
+  implementation("org.apache.commons:commons-dbcp2:2.9.0")
+
+  val relationJdbcDeps: String? = project.findProperty("relationalJdbcDeps") as String?
+  relationJdbcDeps?.let {
+    val dependenciesList = it.split(",")
+    dependenciesList.forEach { dep ->
+      val trimmedDep = dep.trim()
+      if (isValidDep(trimmedDep)) {
+        implementation(trimmedDep)
```

Comment:
repeating here: I'm opposed to conditional dependencies.

How do you envision this to be used?

Reply:
The same way as we do eclipse-link : 
```
./gradlew clean :polaris-quarkus-server:assemble :polaris-quarkus-admin:assemble \
   -PeclipseLinkDeps=org.postgresql:postgresql:42.7.4 \
   -Dquarkus.container-image.tag=postgres-latest \
   -Dquarkus.container-image.build=true \
   --no-build-cache
   ```
   
  now we get which impl to add in class path via parsing the java prop above.

Reply:
Yeah, current EclipseLink UX is far from idea IMHO :sweat_smile: